ca/adminClient.go

```diff
@@ -664,13 +668,13 @@ retry:
 }
 
 // GetExternalAccountKeys returns all ACME EAB Keys from the GET /admin/acme/eab request to the CA.
-func (c *AdminClient) GetExternalAccountKeys(provisionerName string, opts ...AdminOption) ([]*linkedca.EABKey, error) {
+func (c *AdminClient) GetExternalAccountKeys(provisionerName, reference string, opts ...AdminOption) ([]*linkedca.EABKey, error) {
```

Comment:
@dopey: had another look at this. This also makes use of a call to `List()` to loop through all entries and could thus also go OOM. There will likely be less EAB credentials than authorizations, though, so I don't think the issue will happen soon. This could probably also benefit from the `Iterate()` approach described in #767.